components/renku_data_services/app_config/config.py

```diff
@@ -156,6 +159,22 @@ def from_env(cls, prefix: str = "", namespace: str = "") -> "BuildsConfig":
         vscodium_python_run_image = os.environ.get(f"{prefix}BUILD_VSCODIUM_PYTHON_RUN_IMAGE")
         build_strategy_name = os.environ.get(f"{prefix}BUILD_STRATEGY_NAME")
         push_secret_name = os.environ.get(f"{prefix}BUILD_PUSH_SECRET_NAME")
+        buildrun_retention_after_failed_seconds = int(
+            os.environ.get(f"{prefix}BUILD_RUN_RETENTION_AFTER_FAILED_SECONDS") or "0"
```

Comment:
```suggestion
            os.environ.get(f"{prefix}BUILD_RUN_RETENTION_AFTER_FAILED_SECONDS","0")
```

Reply:
This does not work when the env var is set to `""` (empty string):
```
  File "/app/env/lib/python3.12/site-packages/renku_data_services/app_config/config.py", line 170, in from_env
    buildrun_retention_after_succeeded_seconds = int(
                                                 ^^^^
ValueError: invalid literal for int() with base 10: ''
```